server/users.go

```diff
@@ -245,6 +242,28 @@ func (s *Service) UpdateUser(w http.ResponseWriter, r *http.Request) {
 	// ValidUpdate should ensure that req.Roles is not nil
 	u.Roles = req.Roles
 
+	// If the request contains a name, it must be the same as the
```

Comment:
When this is documented in swagger you'll mark fields that cannot be updated as "readOnly"